src/Misc/basis.ts

```diff
@@ -493,3 +493,21 @@ function workerFunc(): void {
         return dst;
     }
 }
+
+Object.defineProperty(BasisTools, "JSModuleURL", {

```

Comment:
Could we move it near the type declaration ? I wonder if in maintenance we might not miss it when reading back the code ?

Reply:
This is moving to LTS in a month or two